src/main/java/at/hannibal2/skyhanni/SkyHanniMod.kt

```diff
@@ -465,6 +466,7 @@ class SkyHanniMod {
         loadModule(BossbarData)
         loadModule(EntityUtils)
         loadModule(ChatUtils)
+        loadModule(ChatClickActionManager)
```

Comment:
pretty sure this is only needed if the class/object needs to be registered for events or for repo patterns